lmdeploy/pytorch/paging/block_trie.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import heapq
+from typing import Dict, Set
+
+import numpy as np
+
+from lmdeploy.pytorch.messages import SchedulerSequence
+
+from ..config import CacheConfig
+from .block_manager import BaseBlockManager
+
+
+class Node:
+    """node of block trie."""
+
+    def __init__(self,
+                 hash_key: int,
+                 block: int,
+                 tokens: np.ndarray,
+                 num_matched: int = 0):
+        self.hash_key = hash_key
+        self.block = block
+        self.tokens = tokens
+        self.num_matched = num_matched
+        self.children: Dict[int, 'Node'] = dict()
+        self._parent: 'Node' = None
+
+    @property
+    def parent(self):
+        return self._parent
+
+    @parent.setter
+    def parent(self, val: 'Node'):
+        old_parent = self._parent
+        if old_parent is not None:
+            old_parent.children.pop(self.hash_key)
+        if val is not None:
+            val.children[self.hash_key] = self
+        self._parent = val
+
+    def __lt__(self, other):
+        return True
```

Comment:
why always `True` ?

Reply:
We don't care the order of nodes with same priority.